thank-you-2017/index.md

```diff
@@ -0,0 +1,663 @@
+---
+layout: blank
```

Comment:
is there a reason for using a completely blank layout as opposed to having this embedded into the website as a regular page (for example, like the sponsor page or the campaign page)? 
I think that further down the line, when we make changes to the website styling and structure, it will be easier to have this page use one of the existing layouts and show the menu bar, as opposed to being a completely separate, standalone thing.